### ISSUE TITLE
Swift 4's key paths not supported

### ISSUE THREAD

**Issue by vincentisambart** [nicklockwood/SwiftFormat#179]
It seems SwiftFormat doesn't support the new Swift 4 key paths. Running it on the file below:
```swift
import WebKit

let webView = WKWebView(frame: .zero, configuration: WKWebViewConfiguration())
let observation = webView.observe(\.estimatedProgress) { _, _ in }
```
gives me the following error:
```
error: unexpected token \.estimatedProgress) at 4:34
```

**Comment by nicklockwood** [comment]
Thanks, I’ll try to fix asap.

**Comment by nicklockwood** [comment]
Fixed in 0.29.7

**Comment by vincentisambart** [comment]
That was quick.
Thanks a lot!
